### PR TITLE
[BUGFIX] Make viewhelpers PHP 7 compatible

### PR DESCRIPTION
**Pull request by liayn**
Fixes: #57

### CODE REVIEW

File: Classes/ViewHelpers/Security/AssetAccessViewHelper.php [hunk 1]
```diff
@@ -59,9 +60,11 @@ public function render(Folder $folder, File $file = null)
      * @param array $arguments
      * @return bool
      */
-    protected static function evaluateCondition($arguments = null)
+    protected static function evaluateCondition(array $arguments)
```
Comment (fsaris) [inline Classes/ViewHelpers/Security/AssetAccessViewHelper.php]: this signature is different in TYPO3 7 and also in \TYPO3Fluid\Fluid\Core\ViewHelper\AbstractConditionViewHelper

Comment (liayn) [inline Classes/ViewHelpers/Security/AssetAccessViewHelper.php]: Sorry for the mistake. update on its way.

**Comment by frans-beech-it** [comment]
Related #61

**Comment by liayn** [comment]
I repushed that already Frans. Github still shows the change request though.

**Pull request merged**